src/tests/JIT/HardwareIntrinsics/X86_AvxVnniInt16/AvxVnniInt16/AvxVnniInt16SampleTest.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics.X86;
+using System.Runtime.Intrinsics;
+using Xunit;
+
+namespace IntelHardwareIntrinsicTest._AvxVnniInt16
```

Comment:
Were these intentionally added? They look to have dead code and aren't actually testing the APIs in question.

We have other tests which cover the CPUID checks